src/backend/marsha/core/serializers/video.py

```diff
@@ -764,3 +765,98 @@ def get_lti_url(self, obj):
             self.context["request"],
             reverse("video_lti_view", args=[obj.id]),
         )
+
+
+class SharedLiveMediaSerializer(serializers.ModelSerializer):
+    """Serializer to display a shared live media model."""
+
+    class Meta:  # noqa
+        model = SharedLiveMedia
+        fields = (
+            "active_stamp",
+            "id",
+            "is_ready_to_show",
+            "pages",
+            "show_download",
+            "title",
+            "upload_state",
+            "urls",
+            "video",
+        )
+        read_only_fields = (
+            "id",
+            "active_stamp",
+            "is_ready_to_show",
+            "pages",
+            "upload_state",
+            "urls",
+            "video",
+        )
+
+    active_stamp = TimestampField(
+        source="uploaded_on", required=False, allow_null=True, read_only=True
+    )
+    urls = serializers.SerializerMethodField()
+    is_ready_to_show = serializers.BooleanField(read_only=True)
+
+    def create(self, validated_data):
+        """Force the video field to the video of the JWT Token if any.
+
+        Parameters
+        ----------
+        validated_data : dictionary
+            Dictionary of the deserialized values of each field after validation.
+
+        Returns
+        -------
+        dictionary
+            The "validated_data" dictionary is returned after modification.
+
+        """
+        # user here is a video as it comes from the JWT
+        # It is named "user" by convention in the `rest_framework_simplejwt` dependency we use.
+        user = self.context["request"].user
+        # Set the video field from the payload if there is one and the user is identified
+        # as a proper user object through access rights
+        if (
+            self.initial_data.get("video")
+            and user.token.get("user")
```

Comment:
I'm not sure why we look at initial data?

Reply:
Because when you create this resource and you are logged on the site, the `video` data is in the payload. It's not the case when you are connected using a token generated by the LTI view.